### PR TITLE
1 Week Preparation Kit: Day 5

### PR DESCRIPTION
**Pull request by jdalbosco**

### CODE REVIEW

File: preparation-kits/one-week/d5-balanced_brackets.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+IS_BALANCED = 'YES'
+IS_NOT_BALANCED = 'NO'
+
+OPENING_BRACKETS = '({['
+
+def is_matching(opening, closing):
+    if opening == '(': return True if closing == ')' else False
+    if opening == '{': return True if closing == '}' else False
+    if opening == '[': return True if closing == ']' else False
```
Comment (camargodev) [inline preparation-kits/one-week/d5-balanced_brackets.py]:
```suggestion
    return opening+closing in ['()', '[]', '{}']
```

File: preparation-kits/one-week/d5-mock_test.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def pairs(diff, numbers_arr):
+    numbers_set = set(numbers_arr)
+    pairs = 0
+    
+    for number in numbers_arr:
+        if number - diff in numbers_set:
```
Comment (camargodev) [inline preparation-kits/one-week/d5-mock_test.py]:
```suggestion
        if (number - diff) in numbers_set:
```

**Review by camargodev — CHANGES_REQUESTED** [review]

File: preparation-kits/one-week/d5-balanced_brackets.py (hunk 1 above)
Comment (jdalbosco) [reply preparation-kits/one-week/d5-balanced_brackets.py]: _chef's kiss_

**Review by camargodev — APPROVED** [review]

File: preparation-kits/one-week/d5-merge_two_sorted_linked_lists.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# SinglyLinkedList() is given in the challenge.
+
+def mergeLists(head_A, head_B):
```
Comment (jdalbosco) [inline preparation-kits/one-week/d5-merge_two_sorted_linked_lists.py]:
```suggestion
def merge_lists(head_A, head_B):
```

**Pull request merged**